scrnaseq.Rmd

```diff
@@ -846,9 +1205,8 @@ p
 ### Dot plot
 ```{r DEG_plot_dot}
 # Visualises how feature expression changes across different clusters
-p = Seurat::DotPlot(sc, features=genes_example[length(genes_example):1], cols=c("lightgrey", param$col)) +
-  ylab("Cluster")
-p = PlotMystyle(p, title="Dot plot of normalised gene expression data") + 
+p = Seurat::DotPlot(sc, features=genes_example[length(genes_example):1], cols=c("lightgrey", param$col), assay="RNA")
```

Comment:
Is the idea to use "RNA" for all plotting??

Reply:
It can happen that I forget. Fixed.